cos-nvidia-gpu-installer/installer.sh

```diff
@@ -47,13 +47,12 @@ BIN_OUTPUT_DIR="${BASE_DIR}/bin"
 KERNEL_SRC_DIR="/lakitu-kernel"
 
 NVIDIA_DRIVER_DIR="/nvidia"
-NVIDIA_DRIVER_VERSION="375.26"
+NVIDIA_DRIVER_VERSION="375.66"
 
-# Source: https://developer.nvidia.com/cuda-downloads
-NVIDIA_CUDA_URL="https://developer.nvidia.com/compute/cuda/8.0/Prod2/local_installers/cuda_8.0.61_375.26_linux-run"
-NVIDIA_CUDA_MD5SUM="33e1bd980e91af4e55f3ef835c103f9b"
-NVIDIA_CUDA_PKG_NAME="cuda_8.0.61_375.26_linux.run"
-NVIDIA_DRIVER_PKG_NAME="NVIDIA-Linux-x86_64-375.26.run"
+# Source: https://www.nvidia.com/Download/index.aspx?lang=en-us
+NVIDIA_DRIVER_URL="https://us.download.nvidia.com/XFree86/Linux-x86_64/${NVIDIA_DRIVER_VERSION}/NVIDIA-Linux-x86_64-${NVIDIA_DRIVER_VERSION}.run"
+NVIDIA_DRIVER_MD5SUM="9cf2c0a342a088211f25c5d8137097e6"
```

Comment:
While I like the idea of not having to bake new images for driver version changes, let's keep the daemonset as dumb as possible for now. We can do this change in a different PR.